shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1346,6 +1389,18 @@ void WebContents::FindReply(content::WebContents* web_contents,
   Emit("found-in-page", result.GetHandle());
 }
 
+void WebContents::RequestKeyboardLock(content::WebContents* web_contents,
+                                      bool esc_key_locked) {
+  exclusive_access_manager_->keyboard_lock_controller()->RequestKeyboardLock(
```

Comment:
We should allow the browser process to deny this; does this go through existing permissions stuff or do we need to add something new?